Routing.sln

```diff
@@ -57,7 +57,15 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "DispatcherSample", "samples
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.AspNetCore.Dispatcher.Performance", "benchmarks\Microsoft.AspNetCore.Dispatcher.Performance\Microsoft.AspNetCore.Dispatcher.Performance.csproj", "{30AF355D-E3AB-4FF5-8A59-A253AFEBA26A}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Microsoft.AspNetCore.Dispatcher.FunctionalTest", "test\Microsoft.AspNetCore.Dispatcher.FunctionalTest\Microsoft.AspNetCore.Dispatcher.FunctionalTest.csproj", "{32107601-C9BE-467B-894C-C9F2E35F03E4}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.AspNetCore.Dispatcher.FunctionalTest", "test\Microsoft.AspNetCore.Dispatcher.FunctionalTest\Microsoft.AspNetCore.Dispatcher.FunctionalTest.csproj", "{32107601-C9BE-467B-894C-C9F2E35F03E4}"
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "shared", "shared", "{2770E573-3E34-4FCE-A3D5-C6CDD7B9A9CC}"
+	ProjectSection(SolutionItems) = preProject
+		shared\Microsoft.AspNetCore.Routing.UrlMatchingTree.Sources\InboundMatch.cs = shared\Microsoft.AspNetCore.Routing.UrlMatchingTree.Sources\InboundMatch.cs
+		shared\Microsoft.AspNetCore.Routing.UrlMatchingTree.Sources\InboundRouteEntry.cs = shared\Microsoft.AspNetCore.Routing.UrlMatchingTree.Sources\InboundRouteEntry.cs
+		shared\Microsoft.AspNetCore.Routing.UrlMatchingTree.Sources\UrlMatchingNode.cs = shared\Microsoft.AspNetCore.Routing.UrlMatchingTree.Sources\UrlMatchingNode.cs
+		shared\Microsoft.AspNetCore.Routing.UrlMatchingTree.Sources\UrlMatchingTree.cs = shared\Microsoft.AspNetCore.Routing.UrlMatchingTree.Sources\UrlMatchingTree.cs
+	EndProjectSection
```

Comment:
I'm really not sure what this is. We don't do it anywhere else.

Reply:
https://github.com/aspnet/StaticFiles/blob/dev/StaticFiles.sln#L31
These will show up in VS if added here. Can be removed though

Reply:
They should show up regardless though right? They show up under each project where they are used

Reply:
Yes they do. This also adds them to the shared directory which shows up in VS. Not really required so I deleted them